requirements.txt

```diff
@@ -10,3 +10,4 @@ requests==2.9.1
 rfc3987==1.3.4
 webapp2==2.5.2
 WebOb==1.5.1
+elasticsearch>=1.0.0,<2.0.0
```

Comment:
I was following this guidelines: http://elasticsearch-py.readthedocs.org/en/master/